content/shmem_ctx.tex

```diff
@@ -64,6 +64,13 @@ \subsubsection{\textbf{SHMEM\_CTX\_CREATE}}
 
         \apitablerow{\CONST{SHMEM\_CTX\_PRIVATE}}{
             The given context will be used only by the thread that created it.}
+
+        \apitablerow{\CONST{SHMEM\_CTX\_NOSTORE}}{
+            Quiet and fence operations performed on the given context are not
```

Comment:
Ok, here's another attempt:

Quiet and fence operations performed on the given context are not required to enforce completion and ordering of memory store operations.  When ordering of store operations is needed, the application must perform a synchronization operation on a context without SHMEM_CTX_NOSTORE option enabled.